### DISCUSSION TITLE
supervised (learning) = แบบมีผู้สอน?

### DISCUSSION THREAD

**Discussion by bact** [srakrn/interpretable-ml-book-th#5] [Translation QAs]
ฉบับภาษาไทยปัจจุบันใช้แปลคำว่า supervised ว่า "ชนิดถูกกำกับ"
https://github.com/srakrn/interpretable-ml-book-th/blob/master/manuscript/01-introduction.Rmd

คำแปลสำหรับคำนี้โดยทั่วไปน่าจะเป็น "แบบมีผู้สอน" มากกว่า

- supervised แบบมีผู้สอน
- unsupervised แบบไม่มีผู้สอน
- semi-supervised แบบกึ่งมีผู้สอน
- reinforcement แบบเสริมกำลัง

ไม่แน่ใจว่าที่เลือกใช้ว่า "ชนิดถูกกำกับ" มีเหตุผลอะไรพิเศษรึเปล่า (เช่นกลัวไปสับสนกับวิธีแบบ Teacher-Student?)

**Comment by srakrn** [comment]
อันนี้เลือกแปล "ชนิดถูกกำกับ" เพื่อให้ได้เซนส์ของการกำกับข้อมูล (annotation) ครับ, ถ้าด้วยเหตุผลนี้ คิดว่ามี tradeoffs ยังไงบ้างครับ?

**Reply by bact** [reply]
พอมันมาแค่ "กำกับ" ห้วนๆ มันนึกไปถึงพวก regularization ได้ด้วยน่ะครับ

**Reply by srakrn** [reply]
จะใช้ "ป้ายกำกับ" (เช่น "ชนิดมีป้ายกำกับ") ก็รู้สึกว่าคำว่าป้ายกำกับให้ sense ของความเป็น "label" (classification) มากกว่า target แบบอื่น (เช่น regression) หนะครับ
